services/galley/test/integration/API.hs

```diff
@@ -1602,10 +1602,10 @@ postO2OConvOk = do
   alice <- randomUser
   bob <- randomUser
   connectUsers alice (singleton bob)
-  a <- postO2OConv alice bob (Just "chat") <!! const 200 === statusCode
-  c <- postO2OConv alice bob (Just "chat") <!! const 200 === statusCode
-  aId <- assertConv a One2OneConv alice alice [bob] (Just "chat") Nothing
-  cId <- assertConv c One2OneConv alice alice [bob] (Just "chat") Nothing
+  a <- postO2OConv alice bob Nothing <!! const 200 === statusCode
+  c <- postO2OConv alice bob Nothing <!! const 200 === statusCode
+  aId <- assertConv a One2OneConv alice alice [bob] Nothing Nothing
+  cId <- assertConv c One2OneConv alice alice [bob] Nothing Nothing
```

Comment:
Because the conversation name is now unset for 1-1 conversations. Before, it was taken from the connection request, but now that's not used anymore, so it needs to be set explicitly.